.travis.yml

```diff
@@ -1,7 +1,7 @@
+branch:
+    only:
+    - "master"
 language: go
 go_import_path: engo.io/ecs
 go:
-- 1.4
-- 1.5
-- 1.6
-- tip
\ No newline at end of file
+- 1.6.1
```

Comment:
Out of all the comments you could make about this PR; I didn't expect this one. :laughing: 

Will add it.